packages/rum/src/domain/record/observers.spec.ts

```diff
@@ -154,3 +154,142 @@ describe('initFrustrationObserver', () => {
     expect(frustrationsCallbackSpy).not.toHaveBeenCalled()
   })
 })
+
+describe('initStyleSheetObserver', () => {
+  let stopStyleSheetObserver: () => void
+  let styleSheetCallbackSpy: jasmine.Spy<StyleSheetRuleCallback>
+  let styleElement: HTMLStyleElement
+  let styleSheet: CSSStyleSheet
+  const styleRule = '.selector-1 { color: #fff }'
+
+  beforeEach(() => {
+    if (isIE()) {
+      pending('IE not supported')
+    }
+    styleSheetCallbackSpy = jasmine.createSpy()
+    styleElement = document.createElement('style')
+    document.head.appendChild(styleElement)
+    styleSheet = <CSSStyleSheet>styleElement.sheet
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏use a non-null assertion, as it is simpler and shows the intent more clearly
```suggestion
    styleSheet = styleElement.sheet!
```